src/lcd.h

```diff
@@ -45,7 +45,7 @@
 
 #define LCD_CONTRAST_INITIAL  14        //!< initial LCD contrast (0-15)
 #define LCD_BLINK_FRAMES      12        //!< refreshes for each frame @ 48 frames/s
-                                        //!< 12 refreshes -> 4Hz Blink frequency
+//!< 12 refreshes -> 4Hz Blink frequency

```

Comment:
If it's not easy to get alignment here, maybe it would be easier to merge the comment with the preceeding line? Again, no biggie...